lib/serializer.js

```diff
@@ -1,5 +1,8 @@
 'use strict'
 
+// eslint-disable-next-line
+const STR_ESCAPE = /[\u0000-\u001f\u0022\u005c\ud800-\udfff]|[\ud800-\udbff](?![\udc00-\udfff])|(?:[^\ud800-\udbff]|^)[\udc00-\udfff]/
```

Comment:
Did you check with safe-regex2 package if it is a safe regex?

Reply:
It's safe.